distr_tracing/distr_tracing_rn/distr-tracing-rn-2-9.adoc

```diff
@@ -0,0 +1,196 @@
+:_content-type: ASSEMBLY
+[id="distributed-tracing-rn-2-9"]
+= Release notes for {DTProductName} 2.9
+include::_attributes/common-attributes.adoc[]
+:context: distributed-tracing-rn-2-9
+
+toc::[]
+
+include::modules/distr-tracing-product-overview.adoc[leveloffset=+1]
+
+== Component versions in the {DTProductName} 2.9
+
+[options="header"]
+|===
+|Operator |Component |Version
+|{JaegerName}
+|Jaeger
+|1.47.0
+
+|{OTELName}
+|OpenTelemetry
+|0.81.0
+
+|{TempoName}
+|Tempo
+|2.1.1
+|===
+
+== {JaegerName}
+
+=== New features and enhancements
+
+* None.
+
+//=== Technology Preview features
+//not for 2.9
+
+=== Bug fixes
+
+* Before this update, connection was refused due to a missing gRPC port on the `jaeger-query` deployment. This issue was causing the following error: *transport: Error while dialing: dial tcp :16685: connect: connection refused*. With this update, the Jaeger Query gRPC port (16685) is successfully exposed on the Jaeger Query service. (link:https://issues.redhat.com/browse/TRACING-3322[TRACING-3322])
+
+* Before this update, the wrong port was exposed for `jaeger-production-query`, resulting in refused connection. With this update, the issue is fixed by exposing the Jaeger Query gRPC port (16685) on the Jaeger Query deployment. (link:https://issues.redhat.com/browse/TRACING-2968[TRACING-2968])
+
+* Before this update, when deploying Service Mesh on single-node OpenShift clusters in disconnected environments, the Jaeger Pod frequently went into the Pending state. With this update, the issue is fixed. (link:https://issues.redhat.com/browse/TRACING-3312[TRACING-3312])
+
+* Before this update, the Jaeger Operator Pod was restarting with the default memory value due to the *OOMKilled* error. With this update, this issue is fixed by removing the resource limits. (link:https://issues.redhat.com/browse/TRACING-3173[TRACING-3173])
+
+=== Known issues
+
+* Apache Spark is not supported.
+ifndef::openshift-rosa[]
+
+* The streaming deployment via AMQ/Kafka is unsupported on IBM Z and IBM Power Systems.
+endif::openshift-rosa[]
+
+== {TempoName}
+
+:FeatureName: The {TempoName}
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+=== New features and enhancements
+
+This release introduces the following noteworthy enhancements for the {TempoShortName}:
+
+* Support the link:https://operatorframework.io/operator-capabilities/[operator maturity] Level IV, Deep Insights, which enables upgrading, monitoring, and alerting of `TempoStack` instances and the {TempoOperator}.
```

Comment:
```suggestion
* Support the link:https://operatorframework.io/operator-capabilities/[Operator maturity] Level IV, Deep Insights, which enables upgrading, monitoring, and alerting of `TempoStack` instances and the {TempoOperator}.
```

Reply:
Thank you. Already fixed in response to another comment.